src/story_protocol_python_sdk/resources/Royalty.py

```diff
@@ -185,64 +190,70 @@ def payRoyaltyOnBehalf(self, receiver_ip_id, payer_ip_id, token, amount, tx_opti
                 'gas': 2000000,
-                'gasPrice': self.web3.to_wei('300', 'gwei')
+                'gasPrice': current_gas_price
             })
 
             # Sign the transaction using the account object
             signed_txn = self.account.sign_transaction(transaction)
-            logger.info(f"Signed transaction: {signed_txn}")
+            # logger.info(f"Signed transaction: {signed_txn}")
 
             # Send the transaction
             tx_hash = self.web3.eth.send_raw_transaction(signed_txn.rawTransaction)
-            logger.info(f"Transaction hash: {tx_hash.hex()}")
+            # logger.info(f"Transaction hash: {tx_hash.hex()}")
 
             # Wait for transaction receipt with a longer timeout
             tx_receipt = self.web3.eth.wait_for_transaction_receipt(tx_hash, timeout=600)  # 10 minutes timeout
-            logger.info(f"Transaction receipt: {tx_receipt}")
+            # logger.info(f"Transaction receipt: {tx_receipt}")
 
             if tx_options and tx_options.get('wait_for_transaction'):
                 self.web3.eth.wait_for_transaction_receipt(tx_hash)
 
             return {'txHash': tx_hash.hex()}
 
         except Exception as e:
-            logger.error(f"Failed to pay royalty on behalf: {e}")
+            # logger.error(f"Failed to pay royalty on behalf: {e}")
             raise e
     
-    def claimRevenue(self, snapshotIds, child_ip_id, token):
+    def claimRevenue(self, snapshot_ids, child_ip_id, token):
         try:
             # Get the royalty vault address
             proxy_address = self._getRoyaltyVaultAddress(child_ip_id)
-            logger.info(f"The proxy address: {proxy_address}")
+            # logger.info(f"The proxy address: {proxy_address}")
             # print("The proxy addres: ", proxy_address)
 
             # Initialize the IP Royalty Vault client with the proxy address
             ip_royalty_vault_client = IpRoyaltyVaultImplClient(self.web3, contract_address=proxy_address)
 
+            # Fetch the current average gas price from the node plus 10%
+            current_gas_price = int(self.web3.eth.gas_price * 1.1)
+
             # Build the transaction
-            transaction = ip_royalty_vault_client.build_claimRevenueBySnapshotBatch_transaction(snapshotIds, token, {
+            transaction = ip_royalty_vault_client.build_claimRevenueBySnapshotBatch_transaction(snapshot_ids, token, {
                 'from': self.account.address,
                 'nonce': self.web3.eth.get_transaction_count(self.account.address),
                 'gas': 2000000,
-                'gasPrice': self.web3.to_wei('300', 'gwei')
+                'gasPrice': current_gas_price
             })
 
             # Sign the transaction using the account object
             signed_txn = self.account.sign_transaction(transaction)
-            logger.info(f"Signed transaction: {signed_txn}")
+            # logger.info(f"Signed transaction: {signed_txn}")
```

Comment:
for those comment, why don't just remove it?  Or use a different level of logger like logger.debug (I don't if there is one though).

Reply:
Done